delocate/tools.py

```diff
@@ -180,6 +180,9 @@ def _cmd_out_err(cmd):
     return out.split('\n')
 
 
+_LINE0_RE = re.compile(r"^(?: \(architecture .*\))?:(?P<further_report>.*)")
```

Comment:
Would you mind splitting these changes off into a separate PR, after #94 is merged?   Maybe add a comment above this line explaining the "architecture" string?

Reply:
Sure. I'll send a PR after #94 is merged.